### PR TITLE
rename bug action message creator

### PR DESCRIPTION
**Pull request by haonliu**
#### Description of changes

rename bug action message creator to issue filing action message creator
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes.

### CODE REVIEW

File: src/DetailsView/details-view-initializer.ts [hunk 1]
```diff
@@ -143,7 +143,7 @@ if (isNaN(tabId) === false) {
             );
             const dropdownActionMessageCreator = new DropdownActionMessageCreator(telemetryFactory, actionMessageDispatcher);
 
-            const bugActionMessageCreator = new BugActionMessageCreator(
+            const bugActionMessageCreator = new IssueFilingActionMessageCreator(
```
Comment (smoralesd) [inline src/DetailsView/details-view-initializer.ts]: miss the `const` here

Comment (smoralesd) [reply src/DetailsView/details-view-initializer.ts]: And I see the same on other files too.

File: src/common/components/file-issue-details-button.tsx [hunk 2]
```diff
@@ -10,12 +10,12 @@ import { LadyBugSolidIcon } from '../../common/icons/lady-bug-solid-icon';
...
 import { FileIssueDetailsHandler } from '../file-issue-details-handler';
 import { HTMLElementUtils } from '../html-element-utils';
-import { BugActionMessageCreator } from '../message-creators/bug-action-message-creator';
+import { IssueFilingActionMessageCreator } from '../message-creators/issue-filing-action-message-creator';
 import { CreateIssueDetailsTextData } from '../types/create-issue-details-text-data';
 import { FileIssueDetailsDialog } from './file-issue-details-dialog';
 
 export type FileIssueDetailsButtonDeps = {
-    bugActionMessageCreator: BugActionMessageCreator;
+    bugActionMessageCreator: IssueFilingActionMessageCreator;
```
Comment (peterdur) [inline src/common/components/file-issue-details-button.tsx]: Rename this one too?

File: src/common/components/issue-filing-button.tsx [hunk 3]
```diff
@@ -9,14 +9,14 @@ import { IssueFilingServiceProvider } from '../../issue-filing/issue-filing-serv
...
-import { BugActionMessageCreator } from '../message-creators/bug-action-message-creator';
+import { IssueFilingActionMessageCreator } from '../message-creators/issue-filing-action-message-creator';
 import { FileIssueClickService } from '../telemetry-events';
 import { CreateIssueDetailsTextData } from '../types/create-issue-details-text-data';
 import { IssueFilingNeedsSettingsContentProps, IssueFilingNeedsSettingsContentRenderer } from '../types/issue-filing-needs-setting-content';
 import { IssueFilingServiceProperties, UserConfigurationStoreData } from '../types/store-data/user-configuration-store';
 
 export type IssueFilingButtonDeps = {
-    bugActionMessageCreator: BugActionMessageCreator;
+    bugActionMessageCreator: IssueFilingActionMessageCreator;
```
Comment (peterdur) [inline src/common/components/issue-filing-button.tsx]: and rename this one...

File: src/tests/unit/tests/common/components/file-issue-details-button.test.tsx [hunk 4]
```diff
@@ -23,7 +23,7 @@ describe('FileIssueDetailsButtonTest', () => {
             .returns(() => 'buildText')
             .verifiable();
 
-        const bugActionMessageCreatorMock = Mock.ofType(BugActionMessageCreator);
+        const bugActionMessageCreatorMock = Mock.ofType(IssueFilingActionMessageCreator);
```
Comment (peterdur) [inline src/tests/unit/tests/common/components/file-issue-details-button.test.tsx]: Rename mocks too?

File: src/common/components/file-issue-details-button.tsx (hunk 2 above)
Comment (haonliu) [reply src/common/components/file-issue-details-button.tsx]: done.

File: src/injected/main-window-initializer.ts [hunk 5]
```diff
@@ -50,6 +48,8 @@ import { ShadowUtils } from './shadow-utils';
 import { TargetPageActionMessageCreator } from './target-page-action-message-creator';
 import { WindowInitializer } from './window-initializer';
 
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```
Comment (lisli1) [inline src/injected/main-window-initializer.ts]: Should the copyright headers be at the top of the file?

**Pull request merged**